### ISSUE TITLE
Weights and Biases quick question

### ISSUE THREAD

**Issue by RJVisee44** [experiencor/basic-yolo-keras#262]
Since they framed detection as a YOLO problem, I assume they have weights and biases. (eg. the classic equation for regression: y = xW + b, where W is the weights and b is the bias). Does this just mean that the 'weights' are the weights saved into the h5 file? Does that file also include the bias? 

I'm asking because I am trying to plot these in a histogram to visualize in Tensorboard and I am not sure where exactly to find this information in the code (i.e. where in the code to put tf.summary.histogram("weights", weights)). I tried just setting histogram_freq=1 but Tensorboard doesn't like generators. Still relatively new to tensorflow and tensorboard.

**Comment by experiencor** [comment]
Includes both weight and bias.

**Comment by joubertdamien** [comment]
@experiencor is this case, why in https://github.com/experiencor/keras-yolo2/blob/master/Yolo%20Step-by-Step.ipynb do we have:

x = Conv2D(64, (3,3), strides=(1,1), padding='same', name='conv_2', use_bias=False)(x)
